tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/embed_block.html

```diff
@@ -1 +1,33 @@
-<div class="grid__embed streamfield__embed">{{ value }}</div>
+{% load static %}
+
+<div class="grid__embed streamfield__embed youtube-video-container">
+    {% if is_youtube %}
+        <div class="youtube-video-placeholder">
+            {% if thumbnail_url %}
+                <img src="{{ thumbnail_url }}" alt="Video Thumbnail">
+            {% else %}
+                <img src="{% static 'images/thumbnail-placeholder.svg' %}" alt="Video Thumbnail">
+            {% endif %}
+            <div class="youtube-overlay">
+                <div class="consent-banner">This content is hosted by a third party. By showing the external content you accept the <a href="https://www.youtube.com/t/terms" target="_blank">terms and conditions</a> of YouTube.</div>
+                <div class="button-container">
+                    <button class="consent-button">
+                        Proceed
+                        {# https://tabler.io/icons/icon/player-play #}
```

Comment:
thanks for finding these svgs. I'll check they are ok with design and add them to the sprints file if so - part of front-end work.

Reply:
I'm removing them in the FE work actually - we're changing the functionality a bit at Ben's request and using a checkbox to save prefs and a button in the usual site button style.